src/BloomExe/Collection/BookCollection.cs

```diff
@@ -189,7 +187,34 @@ protected Color CoverColor
 			set { throw new NotImplementedException(); }
 		}
 
-		public static string DownloadedBooksCollectionNameInEnglish = "Books From BloomLibrary.org";
+		public const string DownloadedBooksCollectionNameInEnglish = "Books From BloomLibrary.org";
+		private const string InstalledTemplatesCollectionNameInEnglish = "Templates";
+
+		public bool IsDownloaded { get { return Name == DownloadedBooksCollectionNameInEnglish; } }
+
+		private string FactoryDir
```

Comment:
This simplified things greatly. I had to make ProjectContext.FactoryCollectionsDirectory public, but I think it's worth it. GetInstalledCollectionsDirectory() is finding the place where bloompacks get put, which is different than factory installed stuff.